octavia-cli/octavia_cli/list/connectors_definitions.py

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+import abc
+from enum import Enum
+from typing import Callable, List, Union
+
+import airbyte_api_client
+from airbyte_api_client.api import destination_definition_api, source_definition_api
+
+
+class DefinitionType(Enum):
+    SOURCE = "source"
+    DESTINATION = "destination"
+
+
+class ConnectorsDefinitions(abc.ABC):
+    LIST_LATEST_DEFINITIONS_KWARGS = {"_check_return_type": False}
+
+    @property
+    @abc.abstractmethod
+    def api(
+        self,
+    ) -> Union[source_definition_api.SourceDefinitionApi, destination_definition_api.DestinationDefinitionApi]:  # pragma: no cover
+        pass
+
+    def __init__(self, definition_type: DefinitionType, api_client: airbyte_api_client.ApiClient, list_latest_definitions: Callable):
+        self.definition_type = definition_type
+        self.api_instance = self.api(api_client)
+        self.list_latest_definitions = list_latest_definitions
+
+    @property
+    def fields_to_display(self) -> List[str]:
+        return ["name", "dockerRepository", "dockerImageTag", f"{self.definition_type.value}DefinitionId"]
+
+    @property
+    def response_definition_list_field(self) -> str:
+        return f"{self.definition_type.value}_definitions"
+
+    def _parse_response(self, api_response) -> List[List[str]]:
+        definitions = [
+            [definition[field] for field in self.fields_to_display] for definition in api_response[self.response_definition_list_field]
+        ]
+        return definitions
+
+    @property
+    def latest_definitions(self) -> List[List[str]]:
+        api_response = self.list_latest_definitions(self.api_instance, **self.LIST_LATEST_DEFINITIONS_KWARGS)
+        return self._parse_response(api_response)
+
+    # TODO alafanechere: declare in a specific formatting module because it will probably be reused
```

Comment:
I agree with these comments that these methods should go in a formatting module, they seem very useful to reuse in other commands 👍